pyproject.toml

```diff
@@ -37,6 +37,7 @@ dockerfile = "^3.2.0"
 virtualenv = "^20.23.0"
 watchfiles = "^0.19.0"
 yaspin = "^2.3.0"
+pydantic = "^1.10.9"
```

Comment:
Pydantic is a great addition, lot of things can be moved to it, including TrussConfig.